Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -472,3 +487,4 @@ However, it seems safe to say that the latency differences are not significant a
 * [https://security.googleblog.com/2014/04/speeding-up-and-strengthening-https.html CHACHA20 vs AES-GCM performance study]
 * [http://cvsweb.openbsd.org/cgi-bin/cvsweb/~checkout~/src/usr.bin/ssh/PROTOCOL.certkeys?rev=1.9&content-type=text/plain PROTOCOL.certkeys]
 * [https://wiki.gnupg.org/rfc4880bis rfc44880bis from GnuPG]
+* [https://weakdh.org/ Weak Diffie-Hellman and the Logjam Attack]
```

Comment:
Would also link to https://jbeekman.nl/blog/2015/05/ssh-logjam/ (perhaps on the same line even)